Atomic/scan.py

```diff
@@ -517,3 +526,33 @@ def write_persistent_data(self):
 
         with open(summary_file, 'w') as f:
             json.dump(persistent_data, f, indent=4)
+
+    def harden(self):
+        for target in self.scan_list:
+            util.write_out("Hardening target {}.\n".format(target.id))
+            temp_dir = tempfile.mkdtemp()
+            fix_script = os.path.join(self.results_dir, target.id, "fix.sh")
+
+            shutil.copy(fix_script, temp_dir)
+            dockerfile_path = os.path.join(temp_dir, "Dockerfile")
+            with open(dockerfile_path, "w") as f:
+                f.write("FROM " + target.id + "\n")
+                f.write("COPY fix.sh /\n")
+                f.write("RUN chmod +x /fix.sh; /fix.sh\n")
+            docker_build_cmd = ['docker', 'build', '--no-cache', temp_dir]
```

Comment:
@mpreisler Yes, that was my intention to prevent this situation, although I don't know what will happen without `--no-cache`. I'll check that.